### PR TITLE
Product department page build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built Product department page
- Removed temporary redirect 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-792.demos.haus/careers/
- See that the overview link routes to https://canonical-com-792.demos.haus/careers/product
- Check against [copy doc](https://docs.google.com/document/d/12rAis8nzWUOun0ZEK_L-ZGz1eEMzBgfc7UhN5t9OpJM/edit)
- Test CTAs

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-1870

### CODE REVIEW

**Comment by marinacastejon** [comment]
@mtruj013 thanks, LGTM

File: templates/careers/product.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+{% block content %}
+<section class="p-strip--dark is-deep" style="background-position: 100% 0%; background-size: cover; background-image:url('https://assets.ubuntu.com/v1/78c0cbe3-05_suru2_dark_2K.jpg')">
+  <hr class="is-fixed-width u-no-margin--bottom is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1 class="p-heading--2"><strong>Product</strong></h1>
+    </div>
+    <div class="col-6">
+      <p class="u-sv2">In Product Management, we predict, define, deliver, and promote the coming waves of open source technology. Product Management is both an art and a science &ndash; full of creativity, problem solving, organisation, and leadership. Your experience, instincts, user focus, analytical nature and passion ensure that people across the company look to you as a leader.</p>
```
Comment (petesfrench) [inline templates/careers/product.html:17]:
```suggestion
      <p class="u-sv2">In Product Management, we predict, define, deliver, and promote the coming waves of open source technology. Product Management is both an art and a science &mdash; full of creativity, problem solving, organisation, and leadership. Your experience, instincts, user focus, analytical nature and passion ensure that people across the company look to you as a leader.</p>
```
This should be an em dash, as it is used to separate information in a sentance, [reference](https://www.scribbr.com/language-rules/dashes/#:~:text=The%20shorter%20en%20dash%20(%E2%80%93),a%20break%20in%20a%20sentence.)

File: templates/careers/product.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Working here</h2>
+  </div>
+  <div class="p-strip is-shallow u-no-padding--top">
+    <div class="row">
+      <div class="col-6">
+        <h3 class="p-heading--2">Product Management at Canonical</h3>
+      </div>
+      <div class="col-6">
+        <p>Product Management at Canonical is a nexus between the various parts of the company &ndash; the CEO, engineering, marketing, sales, field sales, support &ndash; and the various audiences that use, contribute to, influence and commercially interact with the open source products under your stewardship.</p>
```
Comment (petesfrench) [inline templates/careers/product.html:39]:
```suggestion
        <p>Product Management at Canonical is a nexus between the various parts of the company &mdash; the CEO, engineering, marketing, sales, field sales, support &mdash; and the various audiences that use, contribute to, influence and commercially interact with the open source products under your stewardship.</p>
```
Same as above

File: templates/careers/product.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    
+  </div>
+  <div class="p-strip is-shallow u-no-padding--top">
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <div class="row">
+      <div class="col-6">
+        <h3 class="p-heading--2">Who you are</h3>
+      </div>
+      <div class="col-6">
+        <p>You are customer obsessed. You over-communicate strategy and vision.  You are strategic and tactical. You use data and your intuition to make decisions.  You love sharing ideas, information and enthusiasm. You partner with and influence the entire organisation &ndash; and are passionate about the future of technology.</p>
```
Comment (petesfrench) [inline templates/careers/product.html:77]:
```suggestion
        <p>You are customer obsessed. You over-communicate strategy and vision.  You are strategic and tactical. You use data and your intuition to make decisions.  You love sharing ideas, information and enthusiasm. You partner with and influence the entire organisation &mdash; and are passionate about the future of technology.</p>
```

File: templates/careers/product.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        <p>Our team has an insatiable intellectual curiosity. We love the question “Why?” and are relentless in finding answers through user interviews, data, and experimentation.</p>
+        <p>In this team, it's important to have a sense of ownership. Every part of the product's success is your responsibility. You will work closely with the CEO and collaborate with every part of the organisation to realise that success &ndash; whether it's marketing, sales, engineering, or any other key stakeholder.</p>
```
Comment (petesfrench) [inline templates/careers/product.html:79]:
```suggestion
        <p>In this team, it's important to have a sense of ownership. Every part of the product's success is your responsibility. You will work closely with the CEO and collaborate with every part of the organisation to realise that success &mdash; whether it's marketing, sales, engineering, or any other key stakeholder.</p>
```

File: templates/careers/product.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        <p>We look for people who are excellent communicators and can speak to and write effectively for everyone - from marketers to engineers to c-suite executives. You communicate in the format the audience best receives information, whether that's written, visually, or with data. Through it all, you communicate with confidence and empathy.</p>
```
Comment (petesfrench) [inline templates/careers/product.html:80]:
```suggestion
        <p>We look for people who are excellent communicators and can speak to and write effectively for everyone &mdash; from marketers to engineers to c-suite executives. You communicate in the format the audience best receives information, whether that's written, visually, or with data. Through it all, you communicate with confidence and empathy.</p>
```

File: templates/careers/product.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        <p>Through it all, you are process-driven and focused. You'll keep our product teams on track by documenting dependencies, planning sprints, and staying ahead of issues that might block our progress.</p>
+    </div>
+    </div>
+  </div>  
+</section>
+
+{% include "shared/_social-icons-section.html" %}
+
+{% with section_classes='p-strip', heading_topic='Product', tag_id='1216', limit='3' %}
+  {% include "shared/_blog-card.html" %}
```
Comment (petesfrench) [inline templates/careers/product.html:91]: From an accessibility standpoint, it is really not clear that the headings of these blog  are links. Especially as other links on the page are blue. At minimum we should be underlining the heading, or preferably matching the style of the other links (although this has its own problem for people with color blindness)
image

**Review by petesfrench — APPROVED** [review]
LGTM, just some minor changes to the copy and an accessibility issue I noticed (probably best to tackle that in a separate issue as not to block this work)

**Pull request merged**